packages/now-next/src/index.ts

```diff
@@ -329,49 +318,34 @@ export const build = async ({
   }
 
   const isLegacy = nextVersionRange && isLegacyNext(nextVersionRange);
-  let shouldRunScript = 'now-build';
-
   debug(`MODE: ${isLegacy ? 'legacy' : 'serverless'}`);
 
   if (isLegacy) {
-    try {
-      await unlinkFile(path.join(entryPath, 'yarn.lock'));
-    } catch (err) {
-      debug('no yarn.lock removed');
-    }
-
-    try {
-      await unlinkFile(path.join(entryPath, 'package-lock.json'));
-    } catch (err) {
-      debug('no package-lock.json removed');
-    }
-
     console.warn(
       "WARNING: your application is being deployed in @vercel/next's legacy mode. http://err.sh/vercel/vercel/now-next-legacy-mode"
     );
 
+    await Promise.all([
+      remove(path.join(entryPath, 'yarn.lock')),
+      remove(path.join(entryPath, 'package-lock.json')),
+    ]);
```

Comment:
We didn't use to error if these failed, plus the debug message is gone. Should we use `allSettled` instead?

Even if `remove` handles the file being missing, not sure if we want to error for other edge cases. I'll leave this to your discretion.

Reply:
The docs for `fs-extra.remove()` - "Removes a file or directory. The directory can have contents. If the path does not exist, *silently does nothing*."

@Timer So I think this is identical 👍 

@TooTallNate Maybe you might win me over into using `fs-extra` now 😉

Reply:
`remove()` is silent if the file doesn't exist. I think throwing for other edge cases is fine, as they essentially should not happen. This code is under the `isLegacy` code path, so I don't think it's executed very often at this point.